src/config/firebase.js

```diff
@@ -1,8 +1,23 @@
-export default {
+// @flow
+
+import CONFIG from 'constants/config'
+
+const PRODUCTION = {
   apiKey: 'AIzaSyAUBUzRsIYz7MGFCWIlIoMzjl_9IMa4bho',
   authDomain: 'daipos.firebaseapp.com',
   databaseURL: 'https://daipos.firebaseio.com',
   projectId: 'daipos',
   storageBucket: 'daipos.appspot.com',
   messagingSenderId: '949375951452',
 }
+
+const STAGING = {
+  apiKey: 'AIzaSyCFyv9JBJolPI8yDaG6xtZv2eRFyqyK4XQ',
```

Comment:
Potentially insecure

Reply:
Removed Surge deployments in favour of Heroku, now we can inject those as env variables. Will do that in a separate PR later.